tools/Common.Netcore.Dependencies.targets

```diff
@@ -3,21 +3,21 @@
   <ItemGroup > 
     <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.20"/>
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.19"/>
-    <PackageReference Include="Microsoft.Azure.PowerShell.Clients.Aks" Version="1.3.1-preview"/>
```

Comment:
This isn't actually making use of the new version anyplace.  Is there anywhere we are using the new version?  Or is this just to enable an RP to use it?

Reply:
@markcowl we're not using it anywhere in this PR -- this functionality should just be enabled for other services to use